samples/tfm/tfm_secure_peripheral/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cmake_minimum_required(VERSION 3.20)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+
+get_target_property(TFM_BINARY_DIR tfm TFM_BINARY_DIR)
```

Comment:
why are you fetching `TFM_BINARY_DIR` here ?
Did you intend to use it below instead of `CMAKE_CURRENT_BINARY_DIR` but changed your mind.

Reply:
configure_file below uses it in the yaml.in file.